packages/devtools_app/lib/src/debugger/flutter/scripts.dart

```diff
@@ -24,12 +24,14 @@ class ScriptPicker extends StatefulWidget {
     @required this.scripts,
     @required this.classes,
     @required this.onSelected,
+    this.libraryFilterFocusNode,
```

Comment:
should this be required?

Reply:
I don't think so, because if it isn't then the text field will just provide its own. The reason to provide it is if you want to control focus from the outside, as we do with the keyboard shortcut.

Reply:
right, but we aren't using the `ScriptPicker` widget anywhere but in the debugger, where we want to make sure we are always recognizing the ctrl + p shortcut. So making this parameter required would guarantee that we don't regress and stop including the focus node.

Reply:
Added focus to the filter text field, even when you open it with the mouse.